npm-debug.log

```diff
@@ -1,27 +1,45 @@
-6 error Darwin 15.5.0
-7 error argv "/Users/jamesboyer/.nvm/versions/node/v6.4.0/bin/node" "/Users/jamesboyer/.nvm/versions/node/v6.4.0/bin/npm" "run" "dev:watch"
-8 error node v6.4.0
-9 error npm  v3.10.3
-10 error missing script: dev:watch
-11 error If you need help, you may report this error at:
-11 error     <https://github.com/npm/npm/issues>
-12 verbose exit [ 1, true ]
+4 verbose run-script [ 'prestart', 'start', 'poststart' ]
+5 info lifecycle mix_bank@1.0.0~prestart: mix_bank@1.0.0
+6 silly lifecycle mix_bank@1.0.0~prestart: no script for prestart, continuing
+7 info lifecycle mix_bank@1.0.0~start: mix_bank@1.0.0
+8 verbose lifecycle mix_bank@1.0.0~start: unsafe-perm in lifecycle true
+9 verbose lifecycle mix_bank@1.0.0~start: PATH: /usr/local/lib/node_modules/npm/bin/node-gyp-bin:/Users/apprentice/kereru-2016/Mix-Bank/node_modules/.bin:/usr/local/bin:/Users/apprentice/.rbenv/shims:/usr/local/bin:/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin
+10 verbose lifecycle mix_bank@1.0.0~start: CWD: /Users/apprentice/kereru-2016/Mix-Bank
+11 silly lifecycle mix_bank@1.0.0~start: Args: [ '-c', 'webpack-dev-server' ]
+12 silly lifecycle mix_bank@1.0.0~start: Returned: code: 1  signal: null
+13 info lifecycle mix_bank@1.0.0~start: Failed to exec start script
+14 verbose stack Error: mix_bank@1.0.0 start: `webpack-dev-server`
+14 verbose stack Exit status 1
+14 verbose stack     at EventEmitter.<anonymous> (/usr/local/lib/node_modules/npm/lib/utils/lifecycle.js:242:16)
+14 verbose stack     at emitTwo (events.js:106:13)
+14 verbose stack     at EventEmitter.emit (events.js:191:7)
+14 verbose stack     at ChildProcess.<anonymous> (/usr/local/lib/node_modules/npm/lib/utils/spawn.js:40:14)
+14 verbose stack     at emitTwo (events.js:106:13)
+14 verbose stack     at ChildProcess.emit (events.js:191:7)
+14 verbose stack     at maybeClose (internal/child_process.js:852:16)
+14 verbose stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:215:5)
+15 verbose pkgid mix_bank@1.0.0
+16 verbose cwd /Users/apprentice/kereru-2016/Mix-Bank
+17 error Darwin 14.5.0
+18 error argv "/usr/local/bin/node" "/usr/local/bin/npm" "start"
+19 error node v6.4.0
+20 error npm  v3.10.3
+21 error code ELIFECYCLE
+22 error mix_bank@1.0.0 start: `webpack-dev-server`
+22 error Exit status 1
+23 error Failed at the mix_bank@1.0.0 start script 'webpack-dev-server'.
+23 error Make sure you have the latest version of node.js and npm installed.
+23 error If you do, this is most likely a problem with the mix_bank package,
+23 error not with npm itself.
+23 error Tell the author that this fails on your system:
+23 error     webpack-dev-server
+23 error You can get information on how to open an issue for this project with:
+23 error     npm bugs mix_bank
+23 error Or if that isn't available, you can get their info via:
+23 error     npm owner ls mix_bank
+23 error There is likely additional logging output above.
+24 verbose exit [ 1, true ]
```

Comment:
not ideal, can you remove this file completely please
make sure the gitignore is respecting